lib/Gallery.pm

```diff
@@ -159,5 +160,24 @@ sub resize_image {
 	return $new_name;
 }
 
+sub exifdate {
+	my ($image_path) = @_;
+
+	return 'unused' unless 'date' eq $config->{sort_images_by};
```

Comment:
This feels like an unnecessary restriction; why prevent a caller from using this method in other circumstances?